Tests/AssemblyInitialize.cs

```diff
@@ -29,8 +29,7 @@ public class AssemblyInitialize
     public void SetLogHandler()
     {
         AdjustCurrentDirectory();
-        // save output to file as well
-        Log.LogHandler = new ConsoleLogHandler();
+        Log.LogHandler = new ConsoleErrorLogHandler();
```

Comment:
Maybe we could make this depend on the context, whether we are running on travis or not, think it's useful to still keep all the logs when running locally. Maybe something like a flag https://stackoverflow.com/a/55131551/7334720 will also allow removing it if desired even in travis